BootloaderCommonPkg/BootloaderCommonPkg.dec

```diff
@@ -115,7 +115,13 @@
   gPlatformCommonLibTokenSpaceGuid.PcdSpiIasImage2RegionSize   |0x00000000|UINT32|0x2000019C
 
   ## This PCD controls enabled debug output devcie
-  gPlatformCommonLibTokenSpaceGuid.PcdDebugOutputDeviceMask    |0x00000003|UINT32|0x20000400
+  # Debug Interfaces. BIT0-RAM, BIT1-ISA UART, BIT2 -Not used, BIT3-USB3, BIT4-Serial IO, BIT5-TraceHub
+  gPlatformCommonLibTokenSpaceGuid.PcdDebugInterfaceFlags      |0x00000011|UINT32|0x20000400
+
+  ## This PCD controls deug device number
+  # Serial IO device: 0 - Serial UART0, 1 - Serial UART1, 2 - Serial UART2,
+  # ISA UART:         0 - ISA UART device (base 0x2F8),   1 - ISA UART device (base 0x3F8)
+  gPlatformCommonLibTokenSpaceGuid.PcdDebugDeviceNumber        |0x02|UINT8|0x20000410
```

Comment:
I think it is better to let platform decide the mapping between Debug port number and the actual debug device.  It is hard to cover it in common.   I also think it is better to call PcdDebugPortNumber to align with GetDebugPort/GetDebugPort.

Reply:
Will rename PcdDebugDeviceNumber to PcdDebugPortNumber. For the given debug interface ( RAM, ISA UART, serial IO port, USB, ITH, etc), PcdDebugPortNumber is the debug port number starting from 0. It is very clear to read the code.
The platform could override it if really want to do so.

Reply:
If debug interface has multiple devices set (), what does 0 mean?   is it UART 0 or USB 0 or ITH 0 ?  How do you decide which one you use.   I think debug port number should be a combination of  the debug device and its instance number.

Reply:
Exactly. debug interface should combine with debug port number.